cos-fleetshard-api/model/src/main/java/org/bf2/cos/fleetshard/api/DeploymentSpec.java

```diff
@@ -29,6 +30,7 @@ public class DeploymentSpec {
     private SchemaRegistrySpec schemaRegistry;
     private String desiredState;
     private String secret;
+    private String configMapChecksum;
```

Comment:
What is the purpose to store the configmap checksum in the deployment spec ?

Reply:
To trigger reify when it's contents change, same strategy we use for the integration secret. If you have a better alternative let me know.

Reply:
This should happen anyway as the configmap is among the event sources so every change to the configmap should trigger a reconcile loop but I think the issue here is that we don't go through the augmentation phase ?

Reply:
Reconciliation is triggered but not reify, as we only trigger reify when there are changes specifically to the deployment spec. We could change that behavior as well if you see fit.

Reply:
yeah, that something we must change at some point but not in this pr